doc/sphinx-guides/source/user/dataset-management.rst

```diff
@@ -179,6 +179,43 @@ Additional download options available for tabular data (found in the same drop-d
 - Data File Citation (currently in either RIS, EndNote XML, or BibTeX format); 
 - All of the above, as a zipped bundle. 
 
+Research Code
+-------------
+
+Code files - such as Stata, R, MATLAB, or Python files or scripts - have become a frequent addition to the research data deposited in Dataverse repositories. Research code is typically developed by few researchers with the primary goal of obtaining results, while its reproducibility and reuse aspects are sometimes overlooked. Because several independent studies reported issues trying to rerun research code, please consider the following guidelines if your dataset contains code.
+
+The following are general guidelines applicable to all programming languages.
+
+- Create a README text file in the top-level directory to introduce your project. It should answer questions that reviewers or reusers would likely have, such as how to install and use your code. 
```

Comment:
It might even be a good idea to give a hint that in many cases the code should live in a software-set on its own... This of course can be changed in a future iteration...

Reply:
That's a good point! I think once we have a higher level support for software license (ie in metadata), the guideline should be simultaneously updated, but it doesn't hurt to mention it here!